docs/source/model-registry.rst

```diff
@@ -36,7 +36,7 @@ Annotations and Descriptions
 Model Registry Workflows
 ========================
 If running your own MLflow server, you must use a database-backed backend store in order to access
-the model registry via the UI or API. `See here <../tracking.html#backend-stores>`_ for more information.
+the model registry via the UI or API. `See here <tracking.html#backend-stores>`_ for more information.
```

Comment:
`tracking.html` is in the same directory as `model-registry.html`. `..` is not necessary.